### DISCUSSION TITLE
不同显示缩放情况下，软件显示问题。

### DISCUSSION THREAD

**Discussion by mxqcanis** [Richasy/FantasyCopilot#12] [General]
我的笔记本电脑在150%默认缩放情况下，打开软件，最上面大约5%区域无法显示出来，但是在改到125%缩放的情况下，可以完全显示，在改回150%缩放，就能完全显示软件，但是可以看出软件上下间距缩小了相较于第一次打开。

**Comment by mxqcanis** [comment]
<img width="443" alt="image" src="https://github.com/Richasy/FantasyCopilot/assets/46525389/2c366279-b199-4f0b-a1ca-550007c95d2e">
我发现退出后台重进后又会出现显示问题。

**Comment by Richasy** [comment]
会修复的，可以看这个issue来追踪进度：
#13

**Reply by Richasy** [reply]
该问题已经被修复： #15 
会在之后更新
